src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -85,11 +88,30 @@ class DataPointsResourceV1[F[_]](val requestSession: RequestSession[F])
           }
       }
 
-  def queryStringsById(
-      id: Long,
-      inclusiveStart: Long,
-      exclusiveEnd: Long
-  ): F[Seq[StringDataPoint]] =
+  def queryAggregateById(id: Long, inclusiveStart: Long, exclusiveEnd: Long, granularity: String,
```

Comment:
We should also have `queryAggregateByExternalId`, it's going to be very similar and you may as well copy-paste as necessary.

Reply:
*`queryAggregatesByExternalId` (plural aggregates) :)